Patch by Robert Scheck <robert@fedoraproject.org> for varnish >= 2.1, which adds the
missing (former implicit) linking to libm. And as implicit linking can be dangerous,
this changed, see: http://fedoraproject.org/wiki/Features/ChangeInImplicitDSOLinking

--- bin/varnishtest/Makefile.am		2010-03-24 10:44:13.000000000 +0100
+++ bin/varnishtest/Makefile.am.libm	2010-04-07 22:57:03.000000000 +0200
@@ -25,7 +25,7 @@
 		$(top_builddir)/lib/libvarnish/libvarnish.la \
 		$(top_builddir)/lib/libvarnishcompat/libvarnishcompat.la \
 		$(top_builddir)/lib/libvarnishapi/libvarnishapi.la \
-		${PTHREAD_LIBS}
+		${LIBM} ${PTHREAD_LIBS}
 
 EXTRA_DIST = $(top_srcdir)/bin/varnishtest/tests/*.vtc \
 	$(top_srcdir)/bin/varnishtest/tests/README
